.evergreen/functions.yml

```diff
@@ -678,8 +678,14 @@ functions:
 
           if [[ "$IS_WINDOWS" == "true" ]]; then
             npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=windows_setup --tests time-to-first-query
-            npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=windows_zip --tests auto-update-from
-            npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=windows_msi --tests auto-update-from
+            if [[ "$HADRON_DISTRIBUTION" == "compass-isolated"]]; then
```

Comment:
Part of me wants to keep this functions.yml clean so it specifies the full set for the default compass distribution case and then rather just override the tests parameter inside the cli to exclude the variants that won't work with the distribution.